utilities/decrypt_data/decrypt_multi.py

```
@@ -0,0 +1,237 @@
+                        update_sql = (
+                            f"UPDATE {table} SET " +
+                            ", ".join(set_parts) +
+                            f" WHERE {where_clause}"
+                        )
+
+                        cur.execute(update_sql, params)
+                        logger.info("[%s] Row [%s] updated", table, key_desc)
+                        updated_count += 1
+
+                    except Exception as row_err:
+                        logger.exception("[%s] Row [%s] failed: %s", table, key_desc, row_err)
```

Ruff (0.12.2)

Redundant exception object included in `logging.exception` call

(TRY401)